pkg/v1/webhooks/certificates.go

```diff
@@ -151,6 +151,7 @@ func updateOrCreateTLSSecret(ctx context.Context, k8sclient client.Client, tlsSe
 		if err != nil {
 			return err
 		}
+		return nil
```

Comment:
no need to continue if we got a not found, and successfully created the secret. We can return here.